cmd/du-main.go

```diff
@@ -0,0 +1,187 @@
+/*
+ * MinIO Client (C) 2019 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	neturl "net/url"
+
+	humanize "github.com/dustin/go-humanize"
+	"github.com/fatih/color"
+	"github.com/minio/cli"
+	json "github.com/minio/mc/pkg/colorjson"
+	"github.com/minio/mc/pkg/console"
+	"github.com/minio/mc/pkg/probe"
+)
+
+// du specific flags.
+var (
+	duFlags = []cli.Flag{
+		cli.IntFlag{
+			Name:  "max-depth, d",
+			Usage: "print the total for a folder prefix only if it is N or fewer levels below the command line argument",
+		},
+		cli.BoolFlag{
+			Name:  "human-readable, H",
```

Comment:
Human-readable, can be by default. You can have `-b` to display in bytes. We can also check with AB.

Reply:
OK. I will do the change once its confirmed.

Reply:
@balamurugana Can you confirm this change with AB?